_maps/DKsJungleAdventure/DKs-Jungle-Adventure.yaml

```diff
@@ -170,6 +170,10 @@ ventureCards:
   - 0  # 127 Special arcade adventure! You're invited to play Round the Blocks and Memory Block!
   - 1  # 128 Special bonus! You receive 55 times the number of shops you own in gold coins from the bank!
 changelog:
+  - version: 2
+    added: 
+      - Background
+    changed: Some board spaces changed for better balance.
```

Comment:
I know it's stupid, but the website cant handle this part if it's not a bullet point

Reply:
Oh right, my b. I'll fix in a second. Is it only that one?